deltacat/compute/compactor_v2/steps/merge.py

```diff
@@ -438,36 +445,23 @@ def _materialize(
 
         peak_memory_usage_bytes = get_current_node_peak_memory_usage_in_bytes()
 
+        logger.info(f"Finished merge task...")
+
         return MergeResult(
             materialized_results,
             np.int64(total_deduped_records),
             np.double(peak_memory_usage_bytes),
-            np.double(0.0),
             np.double(time.time()),
         )
 
 
 @ray.remote
-def merge(input: MergeInput) -> MergeResult:
-
-    logger.info(f"Starting merge task...")
-    merge_result, duration = timed_invocation(func=_timed_merge, input=input)
-
-    emit_metrics_time = 0.0
-    if input.metrics_config:
-        emit_result, latency = timed_invocation(
-            func=emit_timer_metrics,
-            metrics_name="merge",
-            value=duration,
-            metrics_config=input.metrics_config,
-        )
-        emit_metrics_time = latency
-
-    logger.info(f"Finished merge task...")
-    return MergeResult(
-        merge_result[0],
-        merge_result[1],
-        merge_result[2],
-        np.double(emit_metrics_time),
-        merge_result[4],
-    )
+def merge(
+    input: MergeInput,
+    metrics_config: Optional[MetricsConfig] = None,
```

Comment:
why can't this be part of input like before?

Reply:
`metrics_config` was never part of the input. We could make it part of the `MergeInput` class, but I feel that doesn't make as much sense, as the `metrics_config` object is not actually used in merge.